Userland/Applications/FileManager/PropertiesWindowHashesHashWidget.gml

```diff
@@ -0,0 +1,51 @@
+@GUI::Widget {
+    fixed_height: 42
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [4, 0, 4, 0]
+        spacing: 4
+    }
+
+    @GUI::Label {
+        name: "hash_kind_label"
+        text: "<Hash Kind>:"
+        tooltip: "Hash Kind"
+        text_alignment: "CenterLeft"
+    }
+
+    @GUI::Widget {
+        fixed_height: 24
+
+        layout: @GUI::HorizontalBoxLayout {
+            margins: [0, 0, 0, 0]
+            spacing: 12
+        }
+
+        @GUI::Button {
+            fixed_width: 24
+            fixed_height: 24
+            tooltip: "Calculate this Hash\nHash Status compared to Clipboard"
+            name: "status_icon_button"
+        }
+
+        @GUI::TextBox {
+            fixed_height: 24
+            name: "hash_text"
+            placeholder: "Hash Value not Calculated"
+            tooltip: "Hash Value"
+            font_type: "FixedWidth"
+            shape: "Box"
+            shadow: "Raised"
```

Comment:
I don't think having these custom styles for this box is bringing much to the table (in addition to being nonstandard)